internal/common/host_utils.go

```diff
@@ -41,3 +49,18 @@ func GetEventSeverityFromHostStatus(status string) string {
 		return models.EventSeverityInfo
 	}
 }
+
+func ValidateHostname(hostname string) error {
```

Comment:
why is it in common? do we expect to use it in other packages?

Reply:
It is currently used from bminventory package, but I didn't want to put it there.
The file is called host_utils and it contains various functions related to host. GetCurrentHostName is also in host_utils.

Reply:
then why not to move to a hostutil package?

Reply:
We can, if you think it is better.  I have no preference, assuming that he whole file is moved to that package

Reply:
OK